### ISSUE TITLE
A11y_VisualStudioCodeClient__Editor_Screen Reader: NVDA Repeats the code with ctrl+ right arrow

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/vscode#186727]
### GitHubTags:
#A11ySev3;#A11yTCS;#A11yMAS;#DesktopApp;#Visual Studio Code Client;#WCAG1.3.1;#DesktopApp;#BM-VisualStudioCodeClient-Win32-Aug2022;#NVDA;#Win32;#Win11;#A11yeDAD;#AINotInScope;

### Environment Details:​
Application Name: Visual Studio Code
Version: 1.80.0-insider (user setup)
V8: 10.8.168.25-electron.0
OS build 25869.1000
NVDA2023.1,

### Repro Steps:
Turn on NVDA
Launch Visual Studio Code
Navigate to any line and press ‘ctrl+ right arrow’.
Verify whether after activating the key NVDA announces the word twice.

### Actual:
NVDA Repeats the code with ctrl+ right arrow. for ex: (query_order[identifier] = "equity") when cursor is going on "]" screen reader is announcing the "right bracket equals" and on pressing Ctrl+right arrow focus is shifting to next text but NVDA is announcing the same information "right bracket  equals".

### Expected:
NVDA should not Repeats the code with ctrl+ right arrow. for ex: (query_order[identifier] = "equity") when cursor is going on "]" screen reader should not announce the "right bracket equals" it should only announce "right bracket" and on pressing Ctrl+right arrow focus shifting to next text "=" then NVDA should announce the "equals" information.

### User Impact:
Screen reader users will miss lead the information if it is being announced by screen reader twice.

### Attachment
![A11y_VisualStudioCodeClient__Editor_Screen Reader](https://github.com/microsoft/vscode/assets/98530069/309cac62-7df3-4e54-a5db-c40ce5bacce8)

**Comment by VSCodeTriageBot** [comment]
This issue has been closed automatically because it needs more information and has not had recent activity. See also our [issue reporting](https://aka.ms/vscodeissuereporting) guidelines.

Happy Coding!
